lib/algo/NonuniformDMD.h

```diff
@@ -121,22 +126,22 @@ class NonuniformDMD : public DMD
     /**
      * @brief Returns a pair of pointers to the state and derivative snapshot matrices
      */
-    std::pair<Matrix*, Matrix*> computeDMDSnapshotPair(const Matrix* snapshots);
+    std::pair<Matrix*, Matrix*> computeDMDSnapshotPair(const Matrix* snapshots) override;
 
     /**
      * @brief Compute phi.
      */
-    void computePhi(struct DMDInternal dmd_internal_obj);
+    void computePhi(struct DMDInternal dmd_internal_obj) override;
```

Comment:
Nit: The "struct"  prefix is not required in these contexts in  C++.

Reply:
The prefix is now deleted.